Automattic/wp-calypso
# Stats: Add highlights to Ads page

#### Proposed Changes

Add highlights section to top of Ads page.

The first pass at getting the highlights working. Uses the existing data from the store and formats it for display.

- Adds new highlights section at the top of the page
- Tooltip should show correct payout/status notices
- Hides old Totals section (below chart)

![AdsHighlightsSnap](https://user-images.githubusercontent.com/40267301/213646405-8f31e0b1-a66d-4f0b-8c23-d8acd759b9e6.png)

Does not yet:

* use correct icon data
* remove the previous implementation

I'd like to address these in follow up PRs.

#### Testing Instructions



* Visit the Calypso Live link.
* View a site with WordAds enabled.
* Visit Stats → Ads and confirm highlights are shown correctly.
* Confirm things work correctly on smaller screen sizes.
* Confirm hover works correctly with the tooltip.

If you need to compare with the old Totals section, you can enable it via the React dev tools. Select the `WordAdsEarnings` component and toggle the `showTotalsSection` property.

![DevTools](https://user-images.githubusercontent.com/40267301/213647731-a290ceb6-71e3-4267-a0ac-348321b28630.png)

The new code is only active on non-production domains so it can safely be merged as is.

#### Pre-merge Checklist





Related to #72045.
